docs/usage-as-operator.md

```diff
@@ -22,6 +22,12 @@ machineImages:
   versions:
   - version: "18.04"
     sourceURL: https://cloud-images.ubuntu.com/bionic/current/bionic-server-cloudimg-amd64.img
+# machineTypesLimits is equivalent to resource limits of pod
+# https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container
+machineTypesLimits:
```

Comment:
Could you please change this API as follows:
```yaml
machineTypes:
- name: standard-1
  limits:
    cpu: "2"
    memory: 8Gi
```
I think it's likely that we would need to specify additional kubevirt-specific info for machine types in the future, so let's have one section `machineTypes` instead of `machineTypesLimits` and then `machineTypesWhatever`.

Reply:
good idea.